editor/src/core/workers/parser-printer/parser-printer-bugs.spec.ts

```diff
@@ -85,6 +86,63 @@ export var App = props => {
       fail(parsedPlainCode)
     }
   })
+  it('does something odd for spread mapping - #1365', () => {
```

Comment:
```suggestion
  it('parses elements that use props spreading - #1365', () => {
```
? (also below)

Reply:
Ah, since this was in a file with `-bugs` in the filename, I wrote that from that viewpoint. But I'm not at all attached to it.